config/authentication.py

```diff
@@ -209,7 +209,7 @@ def authenticate_header(self, request):
         return "Bearer"
 
     def authenticate(self, request):
-        jwt_token = request.META.get("HTTP_AUTHORIZATION")
+        jwt_token = self.get_header(request)
```

Comment:
are you sure we get the Auth header with this method? it will default to the header we use in settings.

Reply:
yes 
<img width="839" alt="image" src="https://github.com/user-attachments/assets/a7bb42ec-8793-4d68-bdc5-aae048807dd1">